src/python/build_management/revisions.py

```diff
@@ -47,6 +47,7 @@
 ]
 DISK_CACHE_SIZE = 1000
 SOURCE_MAP_EXTENSION = '.srcmap.json'
+FIND_BRANCHED_FROM = re.compile(r'Cr-Branched-From:.*\master@{#(\d+)\}')
```

Comment:
is this regex correct? or did you mean something like:

```python
re.compile(r'Cr-Branched-From:.*master@\{#(\d+)\}')
```
(moved the `\` from the `\m` to before the `{`)

Reply:
Good spot!